int32 handle
uint8 deviceRegister
uint16 value
---
bool hasSucceeded